_posts/2017-10-25-rails-girls-berlin-is-growing-up.md

```diff
@@ -0,0 +1,46 @@
+---
+title: "Rails Girls Berlin is growing up"
+date: "2017-10-25"
+coverImage: "IMG_0876b.jpg"
```

Comment:
is this image used? We'll need to update the `_config.yml` as well to include `/posts` collections somehow, and specify the permalink structure there.

Reply:
@shushugah how do i do this? My problem now is that everyhthing is set up BUT no matter what I do I can't see this post. It's linked to on the about page but it goes to a 404 :/